src/components/Toast/index.tsx

```diff
@@ -164,5 +179,3 @@ export default class Toast extends Component {
     );
   }
 }
```

Comment:
This should fix the [broken Toast demo page](https://deploy-preview-257--cauldron-react.netlify.com/components/Toast)
```suggestion
}

Toast.displayName = 'Toast';
```

Reply:
Added this as a static display value instead